corehq/apps/dump_reload/sql/dump.py

```diff
@@ -215,9 +215,15 @@ def get_all_model_iterators_builders_for_domain(model_class, domain, builders, l
         using = [limit_to_db]
 
     for db_alias in using:
-        if not model_class._meta.proxy and router.allow_migrate_model(db_alias, model_class):
-            for builder in builders:
-                yield model_class, builder.build(domain, model_class, db_alias)
+        if model_class._meta.proxy:
+            continue
+
+        master_db = settings.DATABASES[db_alias].get('STANDBY', {}).get('MASTER')
+        if not router.allow_migrate_model(master_db or db_alias, model_class):
```

Comment:
not knowing how `allow_migrate_model` is used ouside this context - would it be advisable to override it and push this logic in there?

Reply:
`router.allow_migrate_model` is used by Django in other places to determine where to run migrations so we can't implement this logic generally. The use case here is quite specific to this use case.